packages/flutter/lib/src/material/outlined_button.dart

```diff
@@ -111,22 +112,7 @@ class OutlinedButton extends ButtonStyleButton {
     required Widget label,
     IconAlignment? iconAlignment,
```

Comment:
We can consider make this a regular constructor so you don't need to pass "addPaddding" information through the default constructor.
```dart

const OutlinedButton({
 ...
}) : _addPadding = false;


const OutlinedButton.icon({
 ...
}) : child = icon != null
          ? _OutlinedButtonWithIconChild(
              iconAlignment: iconAlignment,
              label: label,
              buttonStyle: style,
              icon: icon,
            )
          : label,
      _addPadding = true;

bool _addPadding

```